### DISCUSSION TITLE
Disable extension suggestions

### DISCUSSION THREAD

**Discussion by binarious** [phpactor/phpactor#2293] [Q&A] [answered]
How can I disable phpactor to ask for detected extensions (phpstan for example)?

**Comment by dantleech** [comment]
You... can't currently, I plan to add a feature to disable this behavior, but may I ask why you want to do this?

**Reply by binarious** [reply]
I've used phpactor in neovim for quite some time and since some weeks I get interrupted by these suggestions on every project (once). I work on ~40 php projects and that became a bit tedious.

**Reply by binarious** [reply, marked as answer]
Fixed via https://github.com/phpactor/phpactor/pull/2297. Thank you!
